suite-native/video-assets/src/components/Video.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { View, ActivityIndicator } from 'react-native';
+
+import { Video as ExpoVideo, VideoProps as ExpoVideoProps } from 'expo-av';
+
+import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
+
+import { videos, VideoName } from '../videos';
+
+type VideoProps = ExpoVideoProps & {
+    name: VideoName;
+    aspectRatio?: number;
+};
+
+type VideoStyleProps = {
+    aspectRatio: number;
+};
+
+const videoContainer = prepareNativeStyle((_, { aspectRatio }: VideoStyleProps) => ({
+    flex: 1,
+    aspectRatio,
+    justifyContent: 'center',
+    alignItems: 'center',
```

Comment:
nit: try to group flex items together. While this is not required, try to keep flex properties at top and together for better readability and access to layout props

Reply:
fixed in 95199a91a